services/nginz/conf/common_response.conf

```diff
@@ -0,0 +1,31 @@
+      # Should be overriden when using websockets
```

Comment:
Let's move these configuration values (or includes thereof) to `deploy/services-demo` as they're unrelated to nginz as such, they only make sense in the context of all of wire-server.